apps/api/src/config/typeorm/configuration.ts

```diff
@@ -14,6 +14,8 @@ export default new DataSource({
   password: configService.getOrThrow<string>('DB_PASSWORD'),
   database: configService.getOrThrow<string>('DB_NAME'),
   entities: [],
+  autoLoadEntities: true,
   migrations: ['src/database/migrations/**'],
+  migrationsTableName: 'jobs',
   synchronize: false,
 } as DataSourceOptions);
```

Comment:
Why is migration file(apps/api/src/database/migrations/1714634986974-FileDetails.ts) deleted?

Reply:
Files table was not needed so I deleted the migration file.